README.md

```diff
@@ -1,32 +1,89 @@
+
+### Setting Up Sphinx
+Sphinx is used to generate our project's documentation. To set up Sphinx and generate the documentation locally, you need to follow these steps:
+
+1. **Install Python:** If you don't already have Python installed on your system, you can download it from [python.org](https://www.python.org/)
+2. **Install Sphinx:** Use pip, the Python package manager, to install Sphinx. Run the following command in your terminal:
+```
+pip install sphinx
+```
+3.  **Download Additional Dependencies**
+
+    This project require specific Sphinx extensions listed below:
+    1. ```myst_partser``` to parse our .md to Sphinx compatible file format which is .rst. More information [here](https://pypi.org/project/myst-parser/)
+
+    ```
+    pip install myst-parser
+    ```
+    2. ```sphinxcontrib.openapi``` used for generating documentation from OpenAPI specifications. More information [here](https://pypi.org/project/sphinxcontrib-openapi/)
+
+    ```
+    pip install sphinxcontrib.openapi
+    ```
+    3.  ```sphinx_rtd_theme``` used as the main theme of the documenation.  More information [here](https://pypi.org/project/sphinx-rtd-theme/)
+
+    ```
+    pip install sphinx_rtd_theme
+    ```
+
+### Accessing the Documentation
+To access the project documentation, follow these steps:
+
+1. **Fork the Documentation Repository:** Forking a repository is creating a personal copy of a project you want to work on independently. Follow the steps to fork:
+
+- Click on the [upri-earthquake.github.io](https://github.com/UPRI-earthquake/upri-earthquake.github.io) repository
+- Click "Fork"
+- Select the Destination: When you click "Fork," you'll be prompted to select where you want to fork the repository. Fork it to your user account
+
+Congratulations! You are now have a copy of the project that is under your control, and you can make changes to it as you would with any other Git repository.
+
+2. **Clone the Repository:** If you haven't already, clone the project's repository to your local machine using Git. You can do this by running the following command:
+
+```
+git clone https://github.com/your-project/repository.git
+```
+
+3. **Navigate to the Documentation Directory:** In this repository, the documentation files are stored in a folder named "doc".
+
+```
+cd repository/doc
+```
+
+4. **Open the Documentation:** Depending on the format, you can access the documentation using a web browser for HTML-based documentation, or a text editor for Markdown, RST or other text-based formats.
+
+
+
+## Deploying on GitHub Pages
+To deploy your documentation on GitHub Pages, follow these steps:
+
+1. Configure GitHub Pages: In your Forked GitHub repository, go to the "Settings" tab and scroll down to the "GitHub Pages" section. Set the source branch to "gh-pages" and the "docs" folder. Save the changes.
```

Comment:
Okay, it now works by changing the required directory to use in building the github-pages from /docs to /(root).

Please change the instruction under README. Thanks